reward/stakingManager.go

```diff
@@ -0,0 +1,101 @@
+package reward
+
+import (
+	"github.com/klaytn/klaytn/blockchain"
+	"github.com/klaytn/klaytn/event"
+	"github.com/klaytn/klaytn/params"
+)
+
+const (
+	chainHeadChanSize = 100
+)
+
+type stakingManager struct {
+	abm              *addressBookManager
+	stakingInfoCache *stakingInfoCache
+	governanceHelper governanceHelper
```

Comment:
The type and variable name are the same. How about `govHelper`?

Reply:
+1. As you are already using `abm` for `addressBookManager`, you can also consider using `sic` and `gh` for ` stakingInfoCache` and `governanceHelper`.

Reply:
reflected. gh is used.